lua/autorun/server/bounties.lua

```diff
@@ -0,0 +1,72 @@
+
+hook.Add("PlayerDeath","BountyDeath",function(ply,infl,atk)
+	local bounty = GetPlayerBounty(ply)
+	if bounty > 0 and ply != atk and type(atk) == "Player" then
+		SetPlayerBounty(ply,0)
+		atk:PS_GivePoints(bounty)
+		BotSayGlobal("[edgy]"..atk:Nick().." [fbc]has claimed [gold]"..ply:Nick().."'s [fbc]bounty of [rainbow]"..bounty.." [fbc]points!")
+	end
+end)
+
+function GetPlayerBounty(ply)
+	if ply.bounty == nil then
+		ply.bounty = tonumber(ply:GetPData("bounty",0))
+	end
+	return ply.bounty
+end
+
+function SetPlayerBounty(ply,bounty)
+	ply:SetPData("bounty",bounty)
+	ply.bounty = bounty
+end
+
+function AddBounty(ply,target,amount)
```

Comment:
Can you make it so every player can only set max 10mil of bounties per server restart? Just make a table of steamid->amount set so far